packages/pretty-format/README.md

```diff
@@ -69,7 +69,7 @@ console.log(prettyFormat(onClick, options));
 | key                   | type      | default    | description                                             |
 | :-------------------- | :-------- | :--------- | :------------------------------------------------------ |
 | `callToJSON`          | `boolean` | `true`     | call `toJSON` method (if it exists) on objects          |
-| `compareKeys`         | `function`| `undefined`| compare function used when sorting object keys          |
+| `compareKeys`         | `function|null`| `undefined`| compare function used when sorting object keys, `null` can be used to skip over sorting |
```

Comment:
table should be realigned

Reply:
done 👍

Reply:
same for the other one 😀 

there's also a type error here, but I guess you might know that already? (I need to read through the issues here, I've forgotten all context)

Reply:
I've fixed both - let me know if the solution for the types of problem satisfies your needs. The problem was that when `DEFAULT_OPTIONS` were typed as `Options` they included a function type on the `snapshotFormat` property. This was OK for the `pretty-format` lib itself but later on, those `DEFAULT_OPTIONS` were used in the `jest-config` where the function type on that property is not valid. By changing the type to a subtype of `Options` we don't lose any type-safety and we fix the problem because now `DEFAULT_OPTIONS` don't include that function type there and that runtime object just doesn't include `snapshotFormat` property.